bundles/org.openhab.binding.magentatv/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,224 @@
+	<channel-group-type id="program">
+		<label>Program Information</label>
+		<description>Information on the running program</description>
+		<channels>
+			<channel id="title" typeId="system.media-title"/>
+			<channel id="text" typeId="programText"/>
+			<channel id="start" typeId="programStart"/>
+			<channel id="duration" typeId="programDuration"/>
+			<channel id="position" typeId="programPosition"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="status">
+		<label>Play Status</label>
+		<description>Status information on media play</description>
+		<channels>
+			<channel id="channelCode" typeId="channelCode"/>
+			<channel id="runStatus" typeId="runStatus"/>
+			<channel id="playMode" typeId="playMode"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-type id="channelNumber">
+		<item-type>Number</item-type>
+		<label>Channel</label>
+		<description>Send channel number to switch program</description>
+		<state min="1" max="999" step="1"></state>
+	</channel-type>
+	<channel-type id="channelCode" advanced="true">
+		<item-type>Number</item-type>
+		<label>Channel Code</label>
+		<description>Channel code in the channel list</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="runStatus" advanced="true">
+		<item-type>String</item-type>
+		<label>Status</label>
+		<description>Run status</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="playMode">
+		<item-type>String</item-type>
+		<label>Play Mode</label>
+		<description>Play Mode for running program</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programTitle">
+		<item-type>String</item-type>
+		<label>Program</label>
+		<description>Running program</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programText">
+		<item-type>String</item-type>
+		<label>Description</label>
+		<description>Some info on the running program</description>
+		<state readOnly="true" pattern="%s">
+		</state>
+	</channel-type>
+	<channel-type id="programStart">
+		<item-type>DateTime</item-type>
+		<label>Start</label>
+		<description>Program start time</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="programDuration">
+		<item-type>Number:Time</item-type>
+		<label>Duration</label>
+		<description>Duration of the program</description>
+		<state pattern="%d %unit%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="programPosition">
+		<item-type>Number:Time</item-type>
+		<label>Play Position</label>
+		<description>Play Position since program started</description>
+		<state pattern="%d %unit%" readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="key">
+		<item-type>String</item-type>
+		<label>Key</label>
+		<description>Send Key to the Media Receive (POWER/MENU/INFO... - see documentation)</description>
+		<state>
```

Comment:
If this is a command send to the device it should be:
```suggestion
		<command>
```

Reply:
changed